central/cve/datastore/cluster/internal/store/postgres/store.go

```diff
@@ -4,29 +4,26 @@ package postgres
 
 import (
 	"context"
-	"reflect"
 	"time"
 
 	"github.com/gogo/protobuf/proto"
 	"github.com/jackc/pgx/v4"
 	"github.com/jackc/pgx/v4/pgxpool"
-	"github.com/stackrox/rox/central/globaldb"
 	"github.com/stackrox/rox/central/metrics"
 	pkgSchema "github.com/stackrox/rox/central/postgres/schema"
 	"github.com/stackrox/rox/generated/storage"
 	"github.com/stackrox/rox/pkg/logging"
 	ops "github.com/stackrox/rox/pkg/metrics"
 	"github.com/stackrox/rox/pkg/postgres/pgutils"
-	"github.com/stackrox/rox/pkg/postgres/walker"
 )
 
 const (
 	baseTable  = "cluster_cves"
 	countStmt  = "SELECT COUNT(*) FROM cluster_cves"
-	existsStmt = "SELECT EXISTS(SELECT 1 FROM cluster_cves WHERE Id = $1 AND OperatingSystem = $2)"
+	existsStmt = "SELECT EXISTS(SELECT 1 FROM cluster_cves WHERE Id = $1 AND Cve = $2 AND OperatingSystem = $3)"
```

Comment:
unrelated to this PR, but I wonder how cluster cve has an OS?

Reply:
It will not...It will be a separate proto